suprb/rule/fitness.py

```diff
@@ -24,7 +24,7 @@ class VolumeRuleFitness(RuleFitness, metaclass=ABCMeta):
 
     fitness_func_: Callable
 
-    def __init__(self, alpha: float):
+    def __init__(self, alpha: float = 0.85):
```

Comment:
is this a sensible default?

Reply:
I took the value from `suprb/solution/fitness.py` where it is also implemented and had a default value. I did not do an analysis if that default value makes sense

Reply:
then it very likely does not as it is semantically very very different. what were the tuning results of previous runs?

Reply:
I actually read the wrong line. This default value should always get overridden by either `VolumeEmary` or `VolumeWu`. So I think we can put this to any value we want, but perhaps it makes the most sense to put it to `None`. We need to have a default value if we want to tune this, so something needs to be there

Reply:
if it gets overwritten anyway. why would we want to tune it?

Reply:
![grafik](https://github.com/heidmic/suprb/assets/61311706/ad19cb61-1f5f-46ca-87bc-98a221798284)
from Anton's thesis. Using Wu but I would choose the same defaults

Reply:
for emary 0.99 seems a typical value for the error

Reply:
For tuning we either use `PseudoAccuracy`, `VolumeEmary` or `VolumeWu`. `VolumeEmary` and  `VolumeWu` both inherit from `VolumeRuleFitness`, which needs to specify `alpha` in `__init__`. 
In the tuning script, we check if we use `VolumeEmary` or `VolumeWu` and only then add `alpha` to the tuning. It's basically the same reason we need to add defaults to the `CombinedConstraint` `__init__`

Reply:
Then we should put None as the default for VolumeRuleFitness. We want stuff to break if it is configured badly

Reply:
Done